.github/workflows/datadog-synthetics.yml

```
@@ -21,6 +21,7 @@ on:
 
 jobs:
   build:
+    if: ${{ secrets.DD_API_KEY != '' && secrets.DD_APP_KEY != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)